src/store/modules/keycodes/kb-settings.js

```diff
@@ -75,6 +75,76 @@ export default [
     title: 'Swap Ctrl and GUI on both sides (for macOS)',
     width: 1500
   },
+  {
+    name: 'Swap Caps/Esc',
+    code: 'EC_SWAP',
+    title: 'Swap Caps Lock and Escape',
+    width: 1500
```

Comment:
if we're using the same width in a lot of places, you can replace this with a constant.